rocAL/rocAL/source/meta_data/meta_node_resize_mirror_normalize.cpp

```diff
@@ -90,6 +90,11 @@ void ResizeMirrorNormalizeMetaNode::update_parameters(pMetaDataBatch input_meta_
             bb_coords.push_back(coords_buf[j]);
             bb_labels.push_back(labels_buf[j]);
         }
+        // get roi width and height of output image
+        auto img_roi_size = input_meta_data->get_img_roi_sizes_batch()[i];
+        img_roi_size.w = output_roi[i].x2;
+        img_roi_size.h = output_roi[i].y2;
```

Comment:
Are we storing the dst ROIs in x2 & y2?
Even for dst we would have x1,y1 and x2,y2 right?